resources/schema/Operations.md

```diff
@@ -515,7 +517,62 @@ Returns the permissible variables for a given domain and standard
 
 ## record_count
 
-If no 'filter' or 'group' is provided, returns the number of records in the dataset. If 'filter' is provided, returns the number of records in the dataset that contain the values(s) in the corresponding column(s) provided in the filter. If 'group' is provided, returns the number of rows matching each unique set of the grouping variables. If both 'filter' and 'group' are provided, returns the number of records in the dataset that contain the values(s) in the corresponding column(s) provided in the filter that also match each unique set of the grouping variables.
+If no `filter` or `group` is provided, returns the number of records in the dataset. If `filter` is provided, returns the number of records in the dataset that contain the value(s) in the corresponding column(s) provided in the filter. If `group` is provided, returns the number of rows matching each unique set of the grouping variables. If both `filter` and `group` are provided, returns the number of records in the dataset that contain the value(s) in the corresponding column(s) provided in the filter that also match each unique set of the grouping variables.
+
+If `group` is provided, `group_aliases` may also be provided to assign new grouping variable names so that results grouped by the values in one set of grouping variables can be merged onto a dataset according to the same grouping value(s) stored in different set of grouping variables. When both `group` and `group_aliases` are provided, columns are renamed according to corresponding list position (i.e., the 1st column in `group` is renamed to the 1st columnin `group_aliases`, etc.). If there are more columns listed in `group` than in `group_aliases`, only the `group` columns with corresponding `group_aliases` columns will be renamed. If there are more columns listed in `group_aliases` than in `group`, the extra column names in `group_aliases` will be ignored.
```

Comment:
I believe there is a space missing here at columnin

Reply:
Typo corrected here and on [line 43](https://github.com/cdisc-org/cdisc-rules-engine/blob/706-allow-renaming-of-merge-key-for-operations-with-group/resources/schema/Operations.md?plain=1#L43)